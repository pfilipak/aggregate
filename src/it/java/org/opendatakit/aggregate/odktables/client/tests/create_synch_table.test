# Set up
createUser user1

# Test that we can create a new table
createSynchronizedTable user1 people
    name STRING false
    age INTEGER false
    weight DECIMAL false

# Test that we can't create a table that already exists
createSynchronizedTable user1 people
    name STRING false
    age INTEGER false
    weight DECIMAL false

# Test that we can create a table with properties
createSynchronizedTable user1 props {'key':'value','property2':'value'}
	column1 STRING false
	column2 INTEGER true

# Clean up
deleteSynchronizedTable user1 people
deleteSynchronizedTable user1 props
deleteUser user1