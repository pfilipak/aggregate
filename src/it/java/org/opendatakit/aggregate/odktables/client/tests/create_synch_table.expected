# Set up
createUser user1

# Test that we can create a new table
createSynchronizedTable user1 people

# Test that we can't create a table that already exists
createSynchronizedTable user1 people
Exception: org.opendatakit.aggregate.odktables.client.exception.TableAlreadyExistsException: Table with tableID 'people' already exists!

# Test that we can create a table with properties
createSynchronizedTable user1 props {'key':'value','property2':'value'}
	column1 STRING false
	column2 INTEGER true

# Clean up
deleteSynchronizedTable user1 people
deleteSynchronizedTable user1 props
deleteUser user1